src/amazon-sns-sqs-mcp-server/awslabs/amazon_sns_sqs_mcp_server/generator.py

```diff
@@ -58,6 +61,7 @@ def __init__(
         self.tool_configuration = tool_configuration or {}
         self.skip_param_documentation = skip_param_documentation
         self.__validate_tool_configuration()
+        self.config = Config(user_agent_extra=f'MCP/{self.service_name}/{mcp_server_version}')
```

Comment:
suggestion for adding the `awslabs`: 
```suggestion
        self.config = Config(user_agent_extra=f'awslabs/mcp/{self.service_name}/{mcp_server_version}')
```